src/App.ts

```diff
@@ -369,101 +382,45 @@ export default class App {
     }
 
     /* --------------------- Initialize receiver ---------------------- */
```

Comment:
I think we can now remove the comment since that is clear now via the refactored helper `init*` methods 😄

Reply:
👍 Let the code speak for itself!